lib/src/file_util.rs

```diff
@@ -327,7 +328,11 @@ mod platform {
         // note: if developer mode is not enabled the error code will be 1314,
         // ERROR_PRIVILEGE_NOT_HELD
 
-        symlink_file(original, link)
+        // Symlink targets use forward slash `/` as a path separator, so we
+        // need to convert these to backslashes `\` on Windows. Splitting and
+        // recombining the path components is a simple and safe way to do this.
+        let path = original.as_ref().iter().collect::<PathBuf>();
```

Comment:
We'll probably need reverse transformation when snapshotting?

Since this is a content rewriting (similar to CRLF<->LF), I would make local_working_copy.rs handle it.

Reply:
Correct me if I'm wrong but shouldn't these symlinks show up as a change without a reverse transformation? I'll double check, but I don't think that's happening right now...